docs/content/_apidocs.mdx

```diff
@@ -8,146 +8,695 @@ These docs aim to cover the entire public surface of the core `dagster` APIs, as
+    <tr>
+      <td>
+        <a href="/_apidocs/definitions">Definitions</a> (Code locations)
+      </td>
+      <td>
+        APIs to collect definitions so that tools like the Dagster CLI or
+        Dagster UI can load them as{" "}
+        <a href="/concepts/code-locations">code locations</a>.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/resources">Resources</a>
+      </td>
+      <td>
+        APIs to define <a href="/concepts/resources">resources</a>, which are
+        typically used to model external services, tools, and storage for use
+        within jobs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/config">Config</a>
+      </td>
+      <td>
+        The types available to describe{" "}
+        <a href="/concepts/configuration/config-schema">config schemas</a>.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/loggers">Loggers</a>
+      </td>
+      <td>
+        APIs to define how <a href="/concepts/logging/loggers">logs</a> are
+        stored.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/ops">Ops</a>
+      </td>
+      <td>
+        APIs to define or decorate functions as{" "}
+        <a href="/concepts/ops-jobs-graphs/ops">ops</a>, declare their inputs
+        and outputs, compose ops with each other, as well as the datatypes that
+        op execution can return or yield.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/hooks">Hooks</a>
+      </td>
+      <td>
+        APIs to define Dagster <a href="/concepts/ops-jobs-graphs">hooks</a>,
+        which can be triggered on specific Dagster events.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/graphs">Op graphs</a>
+      </td>
+      <td>
+        APIs to define{" "}
+        <a href="/concepts/ops-jobs-graphs/graphs">
+          a set of interconnected ops
+        </a>
+        .
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/dynamic">Dynamic mapping and collect</a>
+      </td>
+      <td>
+        APIs that allow{" "}
+        <a href="/concepts/ops-jobs-graphs/dynamic-graphs">
+          graph structures to be determined at run time
+        </a>
+        .
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/jobs">Jobs</a>
+      </td>
+      <td>
+        APIs to define jobs that execute a set of ops with specific parameters.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/execution">Execution</a>
+      </td>
+      <td>
+        APIs to{" "}
+        <a href="/concepts/ops-jobs-graphs/job-execution">
+          execute and test jobs and individual ops
+        </a>
+        , the execution context available to ops, job configuration, and the default
+        executors available for executing jobs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <a href="/_apidocs/memoization">Job-level versioning & memoization</a>{" "}
```

Comment:
I think we should move this one entry and the repository entry to the bottom of the list since they are deprecated/legacy

Reply:
Will do!

Reply:
did you decide the Job-level versioning entry should stay up here vs getting moved to the bottom?

Reply:
Oh sorry! Just missed it. I'll move it.